noggin/tests/unit/controller/test_password_reset.py

```diff
@@ -84,7 +84,9 @@ def test_password_changes_user(client, logged_in_dummy_user):
 
 
 @pytest.mark.vcr()
-def test_password_form_with_otp(client, logged_in_dummy_user, dummy_user_with_otp):
+def test_password_form_with_otp(
+    client, logged_in_dummy_user, logged_in_dummy_user_with_otp
```

Comment:
I don't think you need `logged_in_dummy_user` here, since it's already contained in `logged_in_dummy_user_with_otp` (and the test does not use its value).